projects/demo/src/modules/components/notification/notification.template.html

```diff
@@ -26,35 +26,45 @@
     <ng-template pageTab>
         <tui-doc-demo>
             <tui-notification
+                (close)="documentationPropertyClose.emitEvent($event)"
```

Comment:
Did our prettier sorter plugin got updated somehow to different rule? 🤔 

It used to be:
*structural
#ref
boolean
string="value"
[binding]="value"
[(banana)]="box"
(output)="onOutput()"